packages/block-editor/src/components/block-mover/index.js

```diff
@@ -44,24 +44,60 @@ export class BlockMover extends Component {
 	}
 
 	render() {
-		const { onMoveUp, onMoveDown, isFirst, isLast, isDraggable, onDragStart, onDragEnd, clientIds, blockElementId, blockType, firstIndex, isLocked, instanceId, isHidden, rootClientId } = this.props;
+		const { onMoveUp, onMoveDown, orientation, isRTL, isFirst, isLast, isDraggable, onDragStart, onDragEnd, clientIds, blockElementId, blockType, firstIndex, isLocked, instanceId, isHidden, rootClientId } = this.props;
 		const { isFocused } = this.state;
 		const blocksCount = castArray( clientIds ).length;
 		if ( isLocked || ( isFirst && isLast && ! rootClientId ) ) {
 			return null;
 		}
 
+		const getArrowIcon = ( hdir ) => {
+			let arrow;
+			switch ( hdir ) {
+				case 'left':
+					arrow = isRTL ? rightArrow : leftArrow;
+					arrow = orientation === 'horizontal' ? arrow : upArrow;
+					break;
+
+				case 'right':
+					arrow = isRTL ? leftArrow : rightArrow;
+					arrow = orientation === 'horizontal' ? arrow : downArrow;
+					break;
+
+				default:
+					break;
+			}
+			return arrow;
+		};
+
+		const getMovementDirection = ( mdir ) => {
```

Comment:
Same again here. I think calling this `moveDirection` which accepts either `'up'` or `'down'` to match the direction that the block moves in the tree is clearer.